src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const workingState = { ...state };
+  const stateHistory = [];
+
+  for (const action of actions) {
+    // Clear workingState
```

Comment:
Use destructuring for `action`
```suggestion
  for (const action of actions) {
    const { type, ...... } = action;
    // Clear workingState
```